src/common.blocks/card/_theme_navigation/card_theme_navigation.less

```diff
@@ -0,0 +1,33 @@
+.card_theme_navigation {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  padding: 1.5rem 0.7rem;
+  background: #fff;
+  box-shadow: 0 4px 15px rgba(0, 0, 0, 0.2);
+  border-radius: 17px;
+
+  .card__item {
+    color: #000;
```

Comment:
Take attention to Declaration order

http://codeguide.co/#css-declaration-order